app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -170,6 +178,12 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
         return LEANPLUM_ENABLED_LOCALES.contains(locale)
     }
 
+    private val Locale.iso3LanguageOrNull: String?
```

Comment:
Later, this looks like a good candidate for re-use. We could uplift to A-C and add to the support-ktx module.

Reply:
👍